src/lib/cache.ts

```diff
@@ -68,19 +60,17 @@ function createMockClient() {
 }
 
 function createMemcachedClient() {
-  const client = new Memcached(MEMCACHED_URL, {
-    poolSize: MEMCACHED_MAX_POOL,
-  })
-  VerboseEvents.forEach((event) => {
-    client.on(event, () => verbose(`[Cache] ${event}`))
+  console.log("hey")
+  const newClient = new MemcacheClient({
+    server: { server: MEMCACHED_URL, maxConnections: MEMCACHED_MAX_POOL },
```

Comment:
not sure if `maxConnections` is this pool config, there doesn't seem to be any connection pool config in the new lib

Reply:
It [does seem to be](https://github.com/electrode-io/memcache/blob/572495967c2d8f2c7253bace89cf02abc7d97edc/packages/memcache-client/src/lib/memcache-node.ts#L32) an internal maximum on the number of connections made by this client. (I was worried it might be a server option due to its location.)